### DISCUSSION TITLE
Does zed support opening files based on class/interface names?

### DISCUSSION THREAD

**Discussion by goldenratio** [zed-industries/zed#13983] [Feature Requests]
Does zed support opening files based on class/interface names?
In vscode, we can do this by typing `#` in search. https://code.visualstudio.com/docs/editor/editingevolved#_open-symbol-by-name

**Comment by SomeoneToIgnore** [comment]
Would "project symbols: toggle" command help?

![image](https://github.com/zed-industries/zed/assets/2690773/27d9e6f1-5054-4db3-a609-ea9fd8604cb6)

**Reply by goldenratio** [reply]
yep, it works.. thanks
